File: src/main/java/org/impact/helpers/ArrayHelper.java
package org.impact.helpers;

import java.util.ArrayList;

public class ArrayHelper {

    @Deprecated
    public static int measureContentLength(ArrayList<String> arr) {
        int counter = 0;
        for (String var : arr) {
            counter = counter + var.length() + 1;
        }
        return counter;
    }

    public static int measureByteContentLength(ArrayList<byte[]> arr) {
        int counter = 0;
        for (byte[] var : arr) {
            counter = counter + var.length + 1;
        }
        return counter;
    }
}

File: src/main/java/org/impact/helpers/ArrayHelper.kt
package org.impact.helpers

import java.util.ArrayList

object ArrayHelper {
    @Deprecated("")
    fun measureContentLength(arr: ArrayList<String>): Int {
        var counter = 0
        for (`var` in arr) {
            counter = counter + `var`.length + 1
        }
        return counter
    }

    @JvmStatic
    fun measureByteContentLength(arr: ArrayList<ByteArray>): Int {
        var counter = 0
        for (`var` in arr) {
            counter = counter + `var`.size + 1
        }
        return counter
    }
}